server/log_operation_manager.go

```diff
@@ -412,6 +416,14 @@ func (l *LogOperationManager) getLogsAndExecutePass(ctx context.Context) error {
 				start := l.info.TimeSource.Now()
 				count, err := l.logOperation.ExecutePass(ctx, logID, &l.info)
 				if err != nil {
+					// The tree can be frozen underneath the signer goroutine. Don't
+					// flood the log if it is as it's not really an error.
```

Comment:
If a tree moves from `DRAINING` to `FROZEN` while this Goroutine is running, shouldn't there only be a single log (because the next time around the call to `l.getLogIDs()` will skip the `FROZEN` log)?